### PR TITLE
Model student

### PR DESCRIPTION
**Pull request by angadkalra**

### CODE REVIEW

File: vjk/models.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
 
 # Create your models here.
+
+class Student(models.Model):
+	first_name = models.CharField(max_length = 30)
+	last_name = models.CharField(max_length = 30)
+	email = models.EmailField()
+	location = models.CharField(max_length = 100)
+	school = models.CharField(max_length = 50)
+	year_attended = models.IntegerField()
```
Comment (chan-kelv) [inline vjk/models.py]: can use YEAR(4) https://dev.mysql.com/doc/refman/5.7/en/year.html

File: vjk/models.py [hunk 2]
```diff
@@ -1,3 +1,14 @@
 from django.db import models
 
 # Create your models here.
+
+class Student(models.Model):
+	first_name = models.CharField(max_length = 30)
+	last_name = models.CharField(max_length = 30)
+	email = models.EmailField()
+	location = models.CharField(max_length = 100)
+	school = models.CharField(max_length = 50)
```
Comment (chan-kelv) [inline vjk/models.py]: bump to 75 char?

**Review by chan-kelv — CHANGES_REQUESTED** [review]
If have time, please update read me so everyone knows how to access the db

File: vjk/models.py (hunk 1 above)
Comment (angadkalra) [reply vjk/models.py]: We need to use the fields Django supports: https://docs.djangoproject.com/en/1.10/ref/models/fields/#model-field-types

File: vjk/models.py (hunk 2 above)
Comment (angadkalra) [reply vjk/models.py]: will do

Comment (angadkalra) [reply vjk/models.py]: check latest commit

**Pull request merged**